sparta/test/Register/Register_test.cpp

```diff
@@ -929,6 +937,25 @@ int main()
     EXPECT_EQUAL((large->read<uint64_t>()), 0xccddeeeeccddffff);
     std::cout << "Large Register: " << std::endl << large->renderSubtree(-1, true);
 
+    // Test huge register (256b)
+    std::cout << "\nWriting to " << huge << std::endl;
+    EXPECT_EQUAL(huge->getNumBits(), (Register::size_type)256);
+    EXPECT_NOTHROW((huge->write<uint64_t>(0xffffffffffffffff)));
+    std::cout << " now: " << huge << std::endl;
+    EXPECT_NOTHROW((huge->write<uint32_t>(0xeeeeeeee, 1))); // 1-0 (MSB)
+    std::cout << " now: " << huge << std::endl;
+    EXPECT_NOTHROW((huge->write<uint16_t>(0xdddd, 3))); // 3-0 (MSB)
+    std::cout << " now: " << huge << std::endl;
+    EXPECT_NOTHROW((huge->write<uint16_t>(0xdddd, 1))); // 3-2
+    std::cout << " now: " << huge << std::endl;
+    EXPECT_NOTHROW((huge->write<uint8_t>(0xcc, 7))); // 7-0 (MSB)
+    std::cout << " now: " << huge << std::endl;
+    EXPECT_NOTHROW((huge->write<uint8_t>(0xcc, 3))); // 7-4
+    std::cout << " now: " << huge << std::endl;
+    std::cout << " have: " << std::hex << huge->read<uint64_t>() << " expect: " << 0xccddeeeeccddffff << std::endl;
+    EXPECT_EQUAL((huge->read<uint64_t>()), 0xccddeeeeccddffff);
+    std::cout << "Huge Register: " << std::endl << huge->renderSubtree(-1, true);
```

Comment:
This testing is great.. a few changes requested:
1. After each write to the given index, add an `EXPECT_EQUAL` to ensure that index was written properly:
```
EXPECT_NOTHROW((huge->write<uint32_t>(0xeeeeeeee, 1))); // 1-0 (MSB)
EXPECT_EQUAL(huge->read<uint32_t>(1). 0xeeeeeeee);
```
1. The `cout` statements aren't too useful if no one looks at them.  :)  Turn them into `EXPECT_EQUAL` statements.

Reply:
Done.